<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpticalFlowWidget</class>
 <widget class="QWidget" name="OpticalFlowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="cpuRadioButton">
         <property name="text">
          <string>CPU</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="gpuRadioButton">
         <property name="text">
          <string>GPU</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="goodFeaturesTab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>GoodFeatures</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Max. Corners</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="maxCornerSpinBox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Min. Distance</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="minDistSpinBox">
             <property name="minimum">
              <double>0.010000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Block Size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="blockSizeSpinBox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Harris Detector</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="harrisCheckBox">
             <property name="text">
              <string>ON</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>K</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="kFactorSpinBox">
             <property name="minimum">
              <double>0.010000000000000</double>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="opticalFlowLukas">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>OF-LucasKanade</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Win. Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sizeWidthSpinBox"/>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sizeHeightSpinBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Max. Levels</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="maxLevelsSpinBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>FLAGS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="initialFlowRadioButton">
           <property name="text">
            <string>INITIAL_FLOW</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="eigenvalsRadioButton">
           <property name="text">
            <string>LK_MIN_EIGENVALS</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Min. Eigen Threashold:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="thresholdSpinBox">
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="termCritTab">
      <attribute name="title">
       <string>TermCriteria</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QRadioButton" name="countRadioButton">
           <property name="text">
            <string>COUNT</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="epsRadioButton">
           <property name="text">
            <string>EPS</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Max. Count</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="maxCountSpinBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Epsilon</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="epsilonSpinBox"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
